src/content/docs/hero.jsx

```diff
@@ -0,0 +1,24 @@
+// Import necessary libraries
+import React, { useState } from "react";
+import ThreeJSComponent from "./threejs.jsx";
+
+
+// The Hero component
+export default function Hero() {
+    return(
+      <div className="flex flex-col items-center justify-center mt-5">
+        <ThreeJSComponent/>
+
+        {/* Centered YouTube Iframe */}
```

Comment:
Hero should not contain youtube video as well, I believe serena implemented the video frame already